frontend/site/.vitepress/theme/components/InventoryBase.vue

```diff
@@ -183,6 +183,16 @@ const filteredSortedRows = computed(() => {
   return filteredRows.value.slice().sort(compareTrimesters);
 });
 
+watchEffect(() => {
```

Comment:
Could we imagine calling loadMore() at mount time until there are some results directly?